cli/build.go

```diff
@@ -42,20 +42,29 @@ func NewBuildCommand() cli.Command {
 				Usage: "runs drone build with publishing enabled",
 			},
 			cli.StringFlag{
-				Name:  "docker-host",
+				Name:  "docker-host, H",
 				Value: getHost(),
 				Usage: "docker daemon address",
 			},
 			cli.StringFlag{
-				Name:  "docker-cert",
+				Name:  "docker-cert, tlscert",
 				Value: getCert(),
 				Usage: "docker daemon tls certificate",
 			},
 			cli.StringFlag{
-				Name:  "docker-key",
+				Name:  "docker-key, tlskey",
 				Value: getKey(),
 				Usage: "docker daemon tls key",
 			},
+			cli.BoolFlag{
```

Comment:
No, parameter `--tls` are part of [docker daemon parameters](https://docs.docker.com/reference/commandline/cli/#daemon). I use this because it's part of output of comand `docker machine config`:

```shell
$ docker-machine config
--tls --tlscacert=/Users/shawnzhu/.docker/machines/.client/ca.pem --tlscert=/Users/shawnzhu/.docker/machines/.client/cert.pem --tlskey=/Users/shawnzhu/.docker/machines/.client/key.pem -H="tcp://192.168.99.100:2376"
```